src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -29,122 +29,140 @@ redirects:
-
-### Notify the right people at the right time [#workflow-intro]
-
-Once you've used our anomaly detection to identify irregular behavior and correlation logic to group all your relevant issues together, it's important to notify the right people on your team of any potential problems. New Relic created workflows so you can control when and where you want to receive notifications about issues which allow you to funnel the right information to the relevant person or team. Workflows also give you the opportunity to enrich your notifications with additional New Relic data.
-
-To learn how to set up workflows, follow our step by step guide by [in our workflows docs](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows).
-
-<Callout variant="important">
-  Pathways have been replaced by [workflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/).
-  If you still have access to pathways, reach out to your New Relic account representative and ask to be upgraded.
-</Callout>
-
-### Tell us where to send your alerts [#destinations]
-
-After you've created a workflow that connects your team with the information they need,  you need to configure your destinations. Destinations are where we send notifications about your New Relic data. A destination is a unique identifier for a third-party system that you use. Destination settings contain the connection details to integrate with third-party systems and can be used across a variety of tools in New Relic.
-
-To learn how to set up destinations and configure message templates, [explore our notification documentation](/docs/alerts-applied-intelligence/notifications/destinations/).
 
-Note: During a service interruption, Alerts may be unavailable.
+Let's see how alerts work with a practical example. Let's say you want to receive a notification when a host's CPU goes above 80%. The first thing you need to do is [create an alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/) and set a [condition threshold](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/set-thresholds-alert-condition/) that will open an incident with a severity level you configure, critical or warning. Then you can connect that condition to an existing [policy](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) or create a new one. After this, you need to configure a [workflow](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/) to make sure that the right people are notified about it.
```

Comment:
Can you break this out into steps to provide more white space? I think it will be easier for the reader to ingest and flow better. 

1. Create an alert condition
2. Set a condition threshold
3. connect that condition to an existing policy
4. Configure a workflow

Reply:
Yes! Much better!!